firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -531,10 +535,7 @@ private Long getFirstOpenTime() {
   }
 
   public long getTemplateVersionNumber() {
-    if (fetchedConfigsCache.get() != null && fetchedConfigsCache.get().getResult() != null) {
-      return fetchedConfigsCache.get().getResult().getTemplateVersionNumber();
-    }
-    return 1L;
+    return frcMetadata.getLastTemplateVersion();
```

Comment:
Thinking aloud: we might refactor this to just use `frcMetadata.getLastTemplateVersion();` where we need it in the future but `getTemplateVersionNumber()` is already in use, so this maintains the functionality. 👍